packages/cloud-cognitive/src/components/Datagrid/Datagrid.test.js

```diff
@@ -113,6 +703,16 @@ describe(componentName, () => {
     );
   });
 
+  /*
```

Comment:
Maybe we can add a TODO comment to let others know why this is commented out and what's the plan to address it?